src/models/customer-model.js

```diff
@@ -1,27 +1,30 @@
 import mongoose from 'mongoose';
+import uniqueValidator from 'mongoose-unique-validator';
 
 const Schema = mongoose.Schema;
 
-export const CustomerSchema = new Schema(
+const CustomerSchema = new Schema(
     {
         customerName: {
             type: String,
-            required: "Please enter your fullname."
+            required: "Please enter your fullname.",
+            unique: true
         },
         phone: {
```

Comment:
```suggestion
        phone: Number
```

would be better as there is on validation

Reply:
I'll do the neccessary changes here